rust/crd/src/listener.rs

```diff
@@ -198,7 +198,7 @@ mod tests {
     use crate::authentication::ResolvedAuthenticationClasses;
 
     use stackable_operator::builder::ObjectMetaBuilder;
-    use stackable_operator::commons::authentication::tls::TlsAuthenticationProvider;
+    use stackable_operator::commons::authentication::tls::AuthenticationProvider;
     use stackable_operator::commons::authentication::{
         AuthenticationClass, AuthenticationClassProvider, AuthenticationClassSpec,
     };
```

Comment:
Already there but we should group properly whereever we see it :) (did not do cargo fmt)
```suggestion
    use stackable_operator::commons::authentication::{
        AuthenticationClass, AuthenticationClassProvider, AuthenticationClassSpec, tls::AuthenticationProvider
    };
```